spec/lib/percy/capybara/client/builds_spec.rb

```diff
@@ -123,7 +125,7 @@
   end
   describe '#finalize_current_build' do
     it 'finalizes the current build' do
-      build_data = {'data' => {'id' => 123}}
+      build_data = { 'data' => { 'id' => 123 } }
```

Comment:
I'd like to enforce the opposite of this, also. ie. without inner space padding inside hashes.